pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -307,11 +307,11 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
     # (since it will do a futile invocation of gcc (!) to find
     # libuuid, slowing down program startup a lot).
     noldconfigPatch
-  ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform && stdenv.isFreeBSD) [
+  ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform && (stdenv.isFreeBSD || stdenv.hostPlatform.isNetBSD)) [
     # Cross compilation only supports a limited number of "known good"
     # configurations. If you're reading this and it's been a long time
     # since this diff, consider submitting this patch upstream!
-    ./freebsd-cross.patch
```

Comment:
I also made a PR for cpython + freebsd here: https://github.com/python/cpython/pull/124424